### PR TITLE
feat(init): first steps, test homepage

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
In this pull request, I did
- install `codeceptjs` that use `WebdriverIO` as test runner.
- write homepage test and fence test for testing presigned-url and user token.
- test steps are written in `steps` folder
- details things to test are written in `questions` folder
- `portal` folder contains all tests for portal
- `apis` folder contains all tests for REST apis
- running test needs `ACCESS_TOKEN`, `INDEX_USERNAME`, `INDEX_PASSWORD` environment variables.
- then, just run `npm install` and `npm run test`
r @philloooo  @frickjack @giangbui

### CODE REVIEW

File: apis/userTokenTest.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+let expired_access_token = process.env.EXPIRED_ACCESS_TOKEN;
+
+Scenario('test create APIKey success', async(I) => {
+  scope = ['data', 'user'];
+  return expect(I.createAPIKey('/user/credentials/cdis/', scope, access_token)
+    .then(
+      (res) => {
+        key_id = res.body.key_id;
+        api_key = res.body.api_key;
+        console.log(`APIKEY: ${api_key}`);
```
Comment (jbarno) [inline apis/userTokenTest.js]: Good idea to log that?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# To run: docker run -d --name=dataportal -p 80:80 quay.io/cdis/data-portal
```
Comment (frickjack) [inline Dockerfile]: What is this Dockerfile for?

File: codecept.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      "browser": "chrome",
+      "restart": false,
+      "windowSize": "normal",
+      "timeouts": {
+        "script": 6000,
+        "page load": 10000
+      }
+    },
+    "REST": {
+      "endpoint": "https://dev.planx-pla.net",
```
Comment (frickjack) [inline codecept.json]: This should be configurable - we want to run this test against different QA environments

File: package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "version": "0.1.0",
+  "description": "Bionimbus Data Portal",
+  "main": "index.js",
+  "dependencies": {
+    "selenium-standalone": "^6.13.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/uc-cdis/auto-qa.git"
+  },
```
Comment (frickjack) [inline package.json]: git url has changed?

File: run-tests.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+set -e
+npm install
+export ACCESS_TOKEN=$(kubectl exec $(get_pod fence) -it -- fence-create token-create --scopes openid,user,fence,data --type access_token --exp 1800 --username cdis.autotest@gmail.com)
```
Comment (frickjack) [inline run-tests.sh]: Make the kubectl namespace configurable, and the QA environment URL.
Also need to set the INDEXD_ and EXPIRED_ environment variables?

**Pull request merged**